tests/terraform/graph/checks_infra/resource_solvers/exists_solver/test_solver.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+from parameterized import parameterized_class
+
+from tests.terraform.graph.checks_infra.test_base import TestBaseSolver
+
+TEST_DIRNAME = Path(__file__).parent
+
+
+@parameterized_class([{"graph_framework": "NETWORKX"}, {"graph_framework": "IGRAPH"}])
+class ExistsSolver(TestBaseSolver):
+    def setUp(self):
```

Comment:
maybe use `pytest` instead of `unittest` for new tests?

Reply:
the parent calss `TestBaseSolver` comes with a couple of convenient functions, which I didn't plan to duplicate, but I will double check what's the effort 🙂